<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QWidget" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="music_lyrics">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
background-image: url(:/myimage/pic/timg.jpg);</string>
   </property>
  </widget>
  <widget class="QSlider" name="music_slider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="music_start">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="music_pause">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>暂停/继续</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_up">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="music_prev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
  </widget>
  <widget class="QPushButton" name="music_next">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>376</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_down">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QSlider" name="music_volume_slider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>29</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="music_comboBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="music_current_time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="music_total_time">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>99:99</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitToMain">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_head">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>376</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>快进</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
